test/config-luatest/upgrade_test.lua

```diff
@@ -118,7 +121,14 @@ g.test_upgrade = function()
         for _, alert in pairs(require("config"):info().alerts) do
             table.insert(messages, alert.message)
         end
+        t.assert_items_exclude(messages, {exp})
```

Comment:
`exp` is a table, so I guess the `messages` array never contains it.

Reply:
Good catch. Didn't notice, that the initial `exp` is changed afterwards. Fixed